### ISSUE TITLE
Trailing commas in argument lists

### ISSUE THREAD

**Issue by SirNickolas** [projectfluent/fluent.js#418]
invalid = { NUMBER("123", ) }

The code above is [correctly parsed](https://github.com/projectfluent/fluent.js/blob/43124372ec8fc3f34ccf3232d53664415ed1917d/fluent-syntax/src/parser.js#L741), although it [shouldn’t be](https://github.com/projectfluent/fluent/blob/94dae4de9dcdbdb3a524191f6382b7886fb5bf31/spec/fluent.ebnf#L72). Trailing commas are an amazingly useful feature, and I’d love to see them in the next version of the spec. But for now they are invalid.

**Comment by Pike** [comment]
The trailing `Argument` is optional, making the trailing comma legal. You tricked me there, I tried t hunt down why they weren't, only to find the conversation that they were, then I dug up the tess, and then re-read the ebnf ;-)

The tests in the spec repo also verify that in

https://github.com/projectfluent/fluent/blob/94dae4de9dcdbdb3a524191f6382b7886fb5bf31/test/fixtures/call_expressions.ftl#L77-L99

That was an explicit decision back in https://github.com/projectfluent/fluent/pull/103#pullrequestreview-121003657.

Closing.
